pipeline_tools/create_envelope.py

```diff
@@ -1,277 +1,416 @@
 #!/usr/bin/env python
-
-import json
 import argparse
-import requests
-from .dcp_utils import get_auth_token, make_auth_header
+import json
+from typing import List
```

Comment:
This import is not being used, can we remove it?

Reply:
In fact, it is used in some docstrings, like this: https://github.com/HumanCellAtlas/pipeline-tools/pull/70/files/39bc791d38668bd5f0ba07e7a22edb9509793ac5#diff-c5363cf30132023e251443047cab1a17R334 , PyCharm/IntelliJ/IDEs are using it to give warnings, and pytest is also using it to make some typing assertions.

Reply:
Thanks, I see where it's used now. So it's required as an import even if it's used in a docstring/comment?

Reply:
Yup, it's required. If we want to adopt type annotations in the future, it will be even more explicitly required.

Reply:
Ok, thanks for the explanation!

Reply:
Thank you!